daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/SpecUtil.scala

```diff
@@ -26,7 +26,7 @@ private[validation] object SpecUtil {
     '∀' -> "forall",
     '→' -> "->",
     '←' -> "<-",
-    '₁' → "_1",
```

Comment:
deprecated and using unicode in one case doesn’t seem helpful.

Reply:
:+1: